dig.php

```diff
@@ -57,184 +58,63 @@ function sig_handler($signo)
 pcntl_signal(SIGHUP,  'sig_handler');
 pcntl_signal(SIGINT,  'sig_handler');
 
-// Repositories to loop for date in
-$app             = realpath(__DIR__);
-$pid_file        = $app . '/pidfile.txt';
-$data_path       = $app   . '/logs/data.json';
-$repos           = $app   . '/repos';
-$git_mozorg      = $repos . '/mozillaorg/';
-$svn_misc        = $repos . '/l10n-misc';
-$git_langchecker = $repos . '/langchecker';
-$git_stores      = $repos . '/appstores';
-
-// Create our data structure
-$update_repos = true;
-
-if (! is_dir($git_mozorg)) {
-    print "Checking our mozilla.org git repository\n";
-    mkdir($repos . '/mozillaorg/');
-    chdir($repos);
-    exec('git clone https://github.com/mozilla-l10n/www.mozilla.org/ mozillaorg');
-} elseif ($update_repos) {
-    chdir($git_mozorg);
-    print "Updating the mozilla.org git repository\n";
-    exec('git checkout master');
-    exec('git pull origin master');
-}
-
-if (! is_dir($svn_misc)) {
-    chdir($repos);
-    print "Checking our l10n-misc svn repository\n";
-    exec('svn co https://svn.mozilla.org/projects/l10n-misc/trunk l10n-misc');
-}
-
-if (! is_dir($git_langchecker)) {
-    chdir($repos);
-    print "Cloning the Langchecker git repository\n";
-    exec('git clone https://github.com/mozilla-l10n/langchecker');
-} elseif ($update_repos) {
-    chdir($git_langchecker);
-    print "Updating the Langchecker git repository\n";
-    exec('git checkout master');
-    exec('git pull origin master');
-}
-
-if (! is_dir($git_stores)) {
-    chdir($repos);
-    print "Cloning the AppStores git repository\n";
-    exec('git clone https://github.com/mozilla-l10n/appstores');
-} elseif ($update_repos) {
-    chdir($git_stores);
-    print "Updating the AppStores git repository\n";
-    exec('git checkout master');
-    exec('git pull origin master');
-}
-
-if (! is_file($git_langchecker . '/composer.phar')) {
-    chdir($git_langchecker);
-    print "Installing composer.\n";
-    exec('curl -sS https://getcomposer.org/installer | php', $output, $err);
-} else {
-    print "Self-updating composer.\n";
-    exec("php $git_langchecker/composer.phar self-update");
-}
-
-if (! is_dir($app . '/logs/')) {
-    mkdir($app . '/logs/');
-}
+// Declare paths
+$app       = realpath(__DIR__);
+$data_path = $app . '/logs/data.json';
 
 if (! file_exists($data_path)) {
     file_put_contents($data_path, json_encode([]));
 }
 
+$data = json_decode(file_get_contents($data_path), true);
```

Comment:
(previous 4 lines) Why not
```
if (file_exists($data_path)) {
    $data = json_decode(file_get_contents($data_path), true);
} else {
    $data = json_encode([]);
}
```

We're going to store the file at the end anyway, no point in storing it empty and then reading it.